### PR TITLE
Update CODEOWNERS file

### PR DESCRIPTION
**Pull request by randimays**
# Objective
Correctly assign code owners for as much `vets-website` code as possible.

Ticket: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15303

## Why?
- Reduces email notification noise from Github
- Increases code quality by having the right teams review the code

## Desired Feedback
Any entries **(for the whole file)** that should be:
- Added
- Removed
- Re-assigned
- Review comments left by others and respond if applicable

**Please review the TODOs and see if you can help clear them up.**

### CODE REVIEW

**Comment by jilladams** [comment]
Nice work! Had no idea there was this much we could do to reduce team burden - thanks @randimays ! I'm going to include a note that we are working on this in ToT today, so new owners are aware this is coming.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -48,39 +53,52 @@ script/github-actions/daily-product-scan/ @department-of-veterans-affairs/qa-sta
...
+src/applications/static-pages/medical-copays-cta @department-of-veterans-affairs/vsa-debt-frontend
 src/applications/combined-debt-portal @department-of-veterans-affairs/vsa-debt-frontend
 src/applications/static-pages/burial-how-do-i-apply-widget @department-of-veterans-affairs/benefits-ce-nondisability
 
 # Public Websites
 
 src/applications/proxy-rewrite @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/va-platform-cop-frontend
 src/platform/site-wide @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/va-platform-cop-frontend
-src/applications/static-pages @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/va-platform-cop-frontend
+src/platform/landing-pages @department-of-veterans-affairs/vsa-public-websites-frontend @department-of-veterans-affairs/vsa-facilities-frontend
```
Comment (jilladams) [inline .github/CODEOWNERS:91]: Not certain about replacing FE with Facilities here vs. adding Facilities on?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -136,12 +157,56 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
 src/applications/mhv/landing-page @department-of-veterans-affairs/vfs-mhv-integration @department-of-veterans-affairs/va-cto-health-products
 src/applications/avs @department-of-veterans-affairs/va-cto-health-products
 
-# VA.gov Product Teams - Forms
+# Dependents Benefits
 
-src/applications/simple-forms @department-of-veterans-affairs/platform-va-product-forms
```
Comment (jilladams) [inline .github/CODEOWNERS:141]: This was removed completely - should it be retained?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -136,12 +157,56 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
...
+
+# TODO: who owns these?
+src/applications/ask-a-question
+src/applications/ask-va
+src/applications/e-folders
+src/applications/messages
+src/applications/mock-sip-form
+src/applications/post-911-gib-status
+src/applications/pre-need
+src/applications/public-outreach-materials
```
Comment (jilladams) [inline .github/CODEOWNERS:195]: This belongs to PW

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -136,12 +157,56 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
...
+src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page  @department-of-veterans-affairs/vfs-vaos-fe @department-of-veterans-affairs/vsa-public-websites-frontend
+
+# Other
+
 src/applications/dhp-connected-devices @department-of-veterans-affairs/digital-health-platform
 src/applications/virtual-agent @department-of-veterans-affairs/orchid
+
+
+
+# TODO: who owns these?
```
Comment (jilladams) [inline .github/CODEOWNERS:187]: Do we know what will happen in terms of autoassigning PR reviewers if we have paths in here without assigned owners? (Aka: do we need to assign owners to each of these lines before merging this PR?)

File: .github/CODEOWNERS (hunk 2 above)
Comment (randimays) [reply .github/CODEOWNERS:141]: Yes! Thanks for the callout. Moved it under Platform Design System

File: .github/CODEOWNERS (hunk 4 above)
Comment (randimays) [reply .github/CODEOWNERS:188]: I think the global Platform assignment at the top (`* @department-of-veterans-affairs/va-platform-cop-frontend`) will pick those up. But I'd like a Platform person to confirm.

File: .github/CODEOWNERS (hunk 1 above)
Comment (randimays) [reply .github/CODEOWNERS:92]: Leaving this here for Facilities folks to review.

File: .github/CODEOWNERS (hunk 4 above)
Comment (randimays) [reply .github/CODEOWNERS:189]: I should add - if these don't get assigned to anyone before we merge, I'll remove them from this file and they should fall into the catch-all anyway.

**Review by longmd — APPROVED** [review]

**Comment by longmd** [comment]
Looks good from the 1010 side of things.

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -16,13 +16,19 @@
...
 src/applications/_mock-form @department-of-veterans-affairs/platform-design-system-fe
 src/applications/burial-poc-v6 @department-of-veterans-affairs/platform-design-system-fe
 src/platform/forms        @department-of-veterans-affairs/platform-design-system-fe @department-of-veterans-affairs/platform-va-product-forms
 src/platform/forms-system @department-of-veterans-affairs/platform-design-system-fe @department-of-veterans-affairs/platform-va-product-forms
+src/applications/simple-forms @department-of-veterans-affairs/platform-va-product-forms
+src/platform/static-pages/simple-forms @department-of-veterans-affairs/platform-design-system-fe @department-of-veterans-affairs/platform-va-product-forms
 script/component-migration @department-of-veterans-affairs/platform-design-system-fe
 src/applications/ds-playground @department-of-veterans-affairs/platform-design-system-fe
 src/applications/ds-v3-playground @department-of-veterans-affairs/platform-design-system-fe
+src/applications/static-pages/wizard @department-of-veterans-affairs/platform-design-system-fe
```
Comment (jamigibbs) [inline .github/CODEOWNERS:31]: If this is forms related, should it be @department-of-veterans-affairs/platform-va-product-forms?

Comment (randimays) [reply .github/CODEOWNERS:31]: Good question - does the DST also need to review? Or `platform-va-product-forms` only?

Comment (jamigibbs) [reply .github/CODEOWNERS:31]: At the moment I think DST is just tagged for visibility for platform form directories but the forms team is generally handling PR reviews related to them. I think it's ok to have both if `/static-pages/wizard` is a platform related form directly but I don't actually know what that is.

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -55,32 +61,46 @@ src/applications/disability-benefits @department-of-veterans-affairs/benefits-te
 src/applications/claims-status @department-of-veterans-affairs/benefits-team-1-frontend
 src/applications/appeals @department-of-veterans-affairs/benefits-team-1-frontend @department-of-veterans-affairs/vsa-authd-exp-frontend
 src/applications/sah @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/coe-access @department-of-veterans-affairs/benefits-team-1-frontend
```
Comment (Mottie) [inline .github/CODEOWNERS:64]: I think `benefits-non-disability` owns COE now

**Review by Mottie — APPROVED** [review]

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -136,12 +159,55 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
...
+src/applications/pre-need
+src/applications/terms-of-use
+src/applications/third-party-app-directory
+src/applications/verify
+src/applications/veteran-id-card
+src/applications/static-pages/analytics
+src/applications/static-pages/ask-va
+src/applications/static-pages/download-1095b
+src/applications/static-pages/i18select
+src/applications/static-pages/shifted-vets-banner
```
Comment (longmd) [inline .github/CODEOWNERS:205]: This one would be public websites, and they could probably remove this, as it was a super old test/initiative that likely was negated by the creation of the new home page.

File: .github/CODEOWNERS (hunk 5 above)
Comment (randimays) [reply .github/CODEOWNERS:31]: 👍  I'll change it to forms team only; that seems to make more sense.

File: .github/CODEOWNERS (hunk 6 above)
Comment (randimays) [reply .github/CODEOWNERS:64]: Got it - thanks!

File: .github/CODEOWNERS [hunk 8]
```diff
@@ -136,12 +159,52 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
 src/applications/mhv/landing-page @department-of-veterans-affairs/vfs-mhv-integration @department-of-veterans-affairs/va-cto-health-products
 src/applications/avs @department-of-veterans-affairs/va-cto-health-products
 
-# VA.gov Product Teams - Forms
+# Dependents Benefits
+src/applications/static-pages/dependency-verification @department-of-veterans-affairs/benefits-ce-dependents
```
Comment (randimays) [inline .github/CODEOWNERS:163]: Why isn't this recognized as a VA Github team?

File: .github/CODEOWNERS [hunk 9]
```diff
@@ -136,12 +159,52 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
...
+src/applications/static-pages/view-modify-dependent @department-of-veterans-affairs/benefits-ce-dependents
+src/applications/static-pages/view-modify-dependents @department-of-veterans-affairs/benefits-ce-dependents
 
-src/applications/simple-forms @department-of-veterans-affairs/platform-va-product-forms
+# Flagship Mobile Reviewers
+src/applications/static-pages/subscription-creators @department-of-veterans-affairs/flagship-mobile-reviewers
```
Comment (randimays) [inline .github/CODEOWNERS:168]: Why isn't this recognized as a VA Github team?

File: .github/CODEOWNERS (hunk 7 above)
Comment (randimays) [reply .github/CODEOWNERS]: Thank you! I'll update.

File: .github/CODEOWNERS [hunk 10]
```diff
@@ -98,6 +118,7 @@ src/applications/caregivers @department-of-veterans-affairs/1010-health-apps-fro
 
 src/applications/hca @department-of-veterans-affairs/1010-health-apps-frontend @department-of-veterans-affairs/benefits-team-1-frontend
 src/applications/ezr @department-of-veterans-affairs/1010-health-apps-frontend @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/letters-mobile-cta @department-of-veterans-affairs/1010-health-apps-frontend
```
[the remark this answers is not in the document]
Comment (randimays) [reply .github/CODEOWNERS:121]: @jilladams Do you know if PW still owns this?

File: .github/CODEOWNERS (hunk 1 above)
Comment (randimays) [reply .github/CODEOWNERS:91]: @maxx1128 @eselkin ^

**Review by jtmst — APPROVED** [review]
I'm pretty sure PW owns src/applications/static-pages/i18select, besides that lgtm!

**Review by chriskim2311 — APPROVED** [review]
LGTM

File: .github/CODEOWNERS [hunk 11]
```diff
@@ -136,12 +159,52 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
...
+
+
+# TODO: who owns these?
+src/applications/ask-a-question
+src/applications/ask-va
+src/applications/e-folders
+src/applications/messages
+src/applications/mock-sip-form
+src/applications/post-911-gib-status
+src/applications/pre-need
```
Comment (jilladams) [inline .github/CODEOWNERS:194]: @BAH-MBS-SelfServ for src/applications/pre-need - per https://dsva.slack.com/archives/CE4304QPK/p1695154166040419?thread_ts=1695153041.910619&cid=CE4304QPK

Comment (randimays) [reply .github/CODEOWNERS]: Thanks @jilladams! Updated

File: .github/CODEOWNERS [hunk 12]
```diff
@@ -136,12 +160,49 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
...
+# VAOS
 
 src/applications/vaos @department-of-veterans-affairs/vfs-vaos-fe
+src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page  @department-of-veterans-affairs/vfs-vaos-fe @department-of-veterans-affairs/vsa-public-websites-frontend
+
+# Other
+
 src/applications/dhp-connected-devices @department-of-veterans-affairs/digital-health-platform
 src/applications/virtual-agent @department-of-veterans-affairs/orchid
+src/applications/pre-need @department-of-veterans-affairs/bah-mbs-selfserv
```
Comment (randimays) [inline .github/CODEOWNERS:185]: Why isn't this recognized as a VA Github team?

File: .github/CODEOWNERS (hunk 10 above)
[the remark this answers is not in the document]
Comment (jilladams) [reply .github/CODEOWNERS:122]: I don't. Have asked Dave: https://dsva.slack.com/archives/C52CL1PKQ/p1695156355685709

File: .github/CODEOWNERS (hunk 9 above)
Comment (jilladams) [reply .github/CODEOWNERS:169]: I suspect there's maybe a repo-level setting, to add this team to vets-website. (Mobile has their own repos.)

File: .github/CODEOWNERS (hunk 10 above)
[the remark this answers is not in the document]
Comment (jilladams) [reply .github/CODEOWNERS:122]: Confirmed: "these should move to 1010 team if they haven't already." https://dsva.slack.com/archives/C52CL1PKQ/p1695156702379509?thread_ts=1695156355.685709&cid=C52CL1PKQ

File: .github/CODEOWNERS (hunk 8 above)
Comment (adamwhitlock1) [reply .github/CODEOWNERS:164]: Maybe the team doesn't have write access to the repo? That would seem strange though

File: .github/CODEOWNERS (hunk 10 above)
[the remark this answers is not in the document]
Comment (randimays) [reply .github/CODEOWNERS:122]: Thanks @jilladams!

File: .github/CODEOWNERS (hunk 9 above)
Comment (randimays) [reply .github/CODEOWNERS:169]: Looking into this with Platform

File: .github/CODEOWNERS (hunk 1 above)
Comment (maxx1128) [reply .github/CODEOWNERS:91]: I think it's better to include the FE team here for public websites since it is code that focuses mainly on the front end.

Comment (randimays) [reply .github/CODEOWNERS:91]: @maxx1128 Do you mean removing Facilities from this?

File: .github/CODEOWNERS [hunk 13]
```diff
@@ -16,13 +16,19 @@
 
 * @department-of-veterans-affairs/va-platform-cop-frontend
 
+
+# Platform Design System
+
 src/applications/_mock-form @department-of-veterans-affairs/platform-design-system-fe
 src/applications/burial-poc-v6 @department-of-veterans-affairs/platform-design-system-fe
```
Comment (jamigibbs) [inline .github/CODEOWNERS:23]: I realize this was here already but I don't think the design system should be assigned to a specific app like this. I'm not sure who is responsible for `burial-poc-v6` though.

Also `src/applications/_mock-form` is an unknown for me. Maybe better for @department-of-veterans-affairs/platform-va-product-forms?

Comment (randimays) [reply .github/CODEOWNERS:23]: Feel free to make any comments about this file, whether they changed or not. :)

I made an update to re-assign those.

File: .github/CODEOWNERS [hunk 14]
```diff
@@ -98,6 +119,7 @@ src/applications/caregivers @department-of-veterans-affairs/1010-health-apps-fro
 
 src/applications/hca @department-of-veterans-affairs/1010-health-apps-frontend @department-of-veterans-affairs/benefits-team-1-frontend
 src/applications/ezr @department-of-veterans-affairs/1010-health-apps-frontend @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/letters-mobile-cta @department-of-veterans-affairs/1010-health-apps-frontend
```
Comment (jilladams) [inline .github/CODEOWNERS:122]: Pivot! https://dsva.slack.com/archives/C52CL1PKQ/p1695232851416249?thread_ts=1695156355.685709&cid=C52CL1PKQ. Moves back to PW.

File: .github/CODEOWNERS [hunk 15]
```diff
@@ -136,12 +160,49 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
...
+src/applications/ask-a-question
+src/applications/ask-va
+src/applications/e-folders
+src/applications/messages
+src/applications/mock-sip-form
+src/applications/post-911-gib-status
+src/applications/terms-of-use
+src/applications/third-party-app-directory
+src/applications/verify
+src/applications/veteran-id-card
```
Comment (jerekshoe) [inline .github/CODEOWNERS:197]: I think you can assign this one to `benefits-team-1`, that's one of my teams applications

File: .github/CODEOWNERS [hunk 16]
```diff
@@ -55,32 +60,48 @@ src/applications/disability-benefits @department-of-veterans-affairs/benefits-te
 src/applications/claims-status @department-of-veterans-affairs/benefits-team-1-frontend
 src/applications/appeals @department-of-veterans-affairs/benefits-team-1-frontend @department-of-veterans-affairs/vsa-authd-exp-frontend
 src/applications/sah @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/nod-cta @department-of-veterans-affairs/benefits-team-1-frontend
+src/applications/static-pages/supplemental-claim @department-of-veterans-affairs/benefits-team-1-frontend
 
 # Benefits Non-Disability
```
[the remark this answers is not in the document]
Comment (jeana-adhoc) [reply .github/CODEOWNERS:66]: `sc/applications/sah` was worked on by a team that's no longer around. The specially adapted housing grant form was worked on by the  'Product Teams - Forms' in their simple-forms directory

**Review by simiadebowale — APPROVED** [review]

File: .github/CODEOWNERS (hunk 16 above)
[the remark this answers is not in the document]
Comment (randimays) [reply .github/CODEOWNERS:66]: @jeana-adhoc Are you saying that the @department-of-veterans-affairs/platform-va-product-forms team should own `src/applications/sah`? Or that `src/applications/sah` is no longer in use?

File: .github/CODEOWNERS (hunk 14 above)
Comment (randimays) [reply .github/CODEOWNERS]: Updated! Thank you

File: .github/CODEOWNERS (hunk 15 above)
Comment (randimays) [reply .github/CODEOWNERS]: Updated! Thank you

File: .github/CODEOWNERS (hunk 1 above)
Comment (maxx1128) [reply .github/CODEOWNERS:92]: @randimays I was thinking more of adding Facilities in addition to FE.

File: .github/CODEOWNERS (hunk 12 above)
Comment (ksantiagoBAH) [reply .github/CODEOWNERS:186]: I added to team to the github workspace should be good now.

Comment (randimays) [reply .github/CODEOWNERS:186]: Thanks @ksantiagoBAH!

File: .github/CODEOWNERS [hunk 17]
```diff
@@ -136,12 +161,48 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
...
+src/applications/messages
+src/applications/mock-sip-form
+src/applications/post-911-gib-status
+src/applications/terms-of-use
+src/applications/third-party-app-directory
+src/applications/verify
+src/applications/static-pages/analytics
+src/applications/static-pages/ask-va
+src/applications/static-pages/download-1095b
+src/applications/static-pages/view-payment-history
```
Comment (jerekshoe) [inline .github/CODEOWNERS:201]: This one should go to `benefits-team-1-frontend`

**Review by rsmithadhoc — APPROVED** [review]

**Review by acrollet — APPROVED** [review]
:octocat: 

(approved for the cto health line)

**Review by maxx1128 — APPROVED** [review]

**Review by amponce — APPROVED** [review]
👍 🚀

**Review by ksantiagoBAH — APPROVED** [review]

**Review by jamigibbs — APPROVED** [review]

**Review by adamwhitlock1 — APPROVED** [review]

File: .github/CODEOWNERS (hunk 16 above)
[the remark this answers is not in the document]
Comment (randimays) [reply .github/CODEOWNERS:70]: @jeana-adhoc Please follow up here when you can. Thanks!

File: .github/CODEOWNERS (hunk 4 above)
Comment (randimays) [reply .github/CODEOWNERS:191]: @rsmithadhoc confirmed any unassigned will fall back to the global Platform assignment at the top of the file (`@department-of-veterans-affairs/va-platform-cop-frontend`) [Slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1695417070193389?thread_ts=1695134841.707539&cid=CBU0KDSB1)

**Pull request merged**